devicetypes/stelpro/stelpro-ki-thermostat.src/stelpro-ki-thermostat.groovy

```diff
@@ -366,7 +367,10 @@ def zwaveEvent(thermostatoperatingstatev1.ThermostatOperatingStateReport cmd) {
 		map.name = "thermostatOperatingState"
 		map.value = operatingState
 
-		if (settings.heatdetails == "No") {
+		// If the user want to see each of the Idle and Heating events in the event history,
+		// Otherwise don't show them more frequently than 5 minutes.
```

Comment:
Just a little bit of nit-picking for this comment and the other two below:
`If the user wants` and `history, otherwise`.